doc/stdenv/stdenv.chapter.md

```diff
@@ -1129,7 +1129,10 @@ They cannot be overridden without rebuilding the package.
 
 If dependencies should be resolved at runtime, use `--suffix` to append fallback values to `PATH`.
 
-There’s many more kinds of arguments, they are documented in `nixpkgs/pkgs/build-support/setup-hooks/make-wrapper.sh` for the `makeWrapper` implementation and in `nixpkgs/pkgs/build-support/setup-hooks/make-binary-wrapper/make-binary-wrapper.sh` for the `makeBinaryWrapper` implementation.
+There’s many more kinds of arguments, they are documented in `nixpkgs/pkgs/build-support/setup-hooks/make-wrapper.sh` for the `makeWrapper` implementation and in `nixpkgs/pkgs/by-name/ma/makeBinaryWrapper/make-binary-wrapper.sh` for the `makeBinaryWrapper` implementation.
+
+While mostly interchangeable, `makeBinaryWrapper` does not support as many arguments as `makeWrapper` does.
+`makeBinaryWrapper` should be used for interpreters, as some systems require script "shebangs" to be binary files.
```

Comment:
There is already a paragraph about when to use `makeBinaryWrapper` below. I would instead add after there:

```
Use `makeWrapper` if you need the wrapper to use shell features (e.g. look up environment variables) at runtime.
```